libmscore/score.cpp

```diff
@@ -2887,24 +2887,36 @@ void Score::padToggle(Pad n, const EditData& ed)
       std::vector<ChordRest*> crs;
 
       if (selection().isSingle()) {
-            // do not allow to add a dot on a full measure rest
-            Element* e = selection().element();
-            if (e && e->isRest()) {
-                  Rest* r = toRest(e);
-                  if (r->isFullMeasureRest())
-                        _is.setDots(0);
-                  }
 
-            // on measure rest, select the first actual rest
-            ChordRest* cr = selection().cr();
-            if (cr && cr->isRest() && cr->measure()->isMMRest()) {
-                  Measure* m = cr->measure()->mmRestFirst();
-                  if (m)
-                        cr = m->findChordRest(m->tick(), 0);
+            Element* selectedElement = selection().element();
+
+            if (!(selectedElement->isNote() || selectedElement->isRest())) {
+
+                  ed.view->startNoteEntryMode();
+                  deselect(selectedElement);
+
```

Comment:
Well, the blank lines (2894 and 2897) should be removed. You aren't separating anything practically with the blank lines.